src/SprykerEco/Zed/Payone/Business/Api/Response/Container/GenericPaymentResponseContainer.php

```diff
@@ -301,7 +306,7 @@ public function setEmail(string $email)
      */
     protected function getPreparedKey($key)
     {
-        $key = preg_replace("/add_paydata\[(.*)\]/", "$1", $key);
+        $key = preg_replace(self::ADD_PAYDATA_PATTERN, "$1", $key);
```

Comment:
It was a code review comment from Alex Galych, he wanted this to be moved to constant. I agree, without the const it is more clear, I'll revert this change.